.github/workflows/dev.yml

```diff
@@ -5,6 +5,7 @@ name: CI
 on:
   push:
     branches: ["dev"]
+    tags: ['v[0-9]*', '[0-9]+.[0-9]+*']  # Match tags that resemble a version
```

Comment:
nit: For consistency move the comment to it's own line above the comment it applies to